diagnostics/model_test/verify_model.py

```diff
@@ -187,21 +187,26 @@ def parse_json():
     np.random.seed(100)
     input_dimension = parameters["input_dimension"]
     input_tensor_name = parameters["input_tensor_name"]
+    # Get random value range
+    rand_val_range = parameters["random_val_range"]
     bs = int(parameters["batch_size"])
 
     assert len(input_dimension) == len(
         input_tensor_name
     ), "input_tensor_name dimension should match input_dimension in json file"
 
-    # Get random value range
-    rand_val_range = parameters["random_val_range"]
+    assert len(input_tensor_name) == len(
+        rand_val_range
+    ), "number of random_val_range should match input_tensor_name in json file"
 
     # Matches the input tensors name with its required dimensions
     input_tensor_dim_map = {}
+    i = 0
     for (dim, name) in zip(input_dimension, input_tensor_name):
         random_input = np.random.randint(
-            rand_val_range, size=[bs] + dim).astype('float32')
+            rand_val_range[i], size=[bs] + dim).astype('float32')
         input_tensor_dim_map[name] = random_input
+        i = i + 1
```

Comment:
Style issue about the extra variable `i`:

Instead of zipping input_dimension and input_tensor_name, and then tracking an `i` variable separately, we can either `for i in range(len(input_dimension)):`
or: zip all three input_dimension, input_tensor_name and rand_val_range

```
for (dim, name, val_range) in zip(input_dimension, input_tensor_name, rand_val_range):
    input_tensor_dim_map[name] = np.random.randint(val_range, size=[bs] + dim).astype('float32')
```

Reply:
Thanks for the comment. I chose to zip all three parameters.